asset_stream_manager/start_command.cc

```diff
@@ -0,0 +1,104 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "asset_stream_manager/start_command.h"
+
+#include <memory>
+
+#include "asset_stream_manager/local_assets_stream_manager_client.h"
+#include "asset_stream_manager/session_management_server.h"
+#include "common/log.h"
+#include "common/path.h"
+#include "common/status_macros.h"
+#include "lyra/lyra.hpp"
+
+namespace cdc_ft {
+
+StartCommand::StartCommand(int* exit_code)
+    : BaseCommand("start",
+                  "Start streaming files from a Windows to a Linux device",
+                  exit_code) {}
+
+StartCommand::~StartCommand() = default;
+
+void StartCommand::RegisterCommandLineFlags(lyra::command& cmd) {
+  verbosity_ = 2;
```

Comment:
It did not store my comments :(
So, again. Could you use a named constant here for `2`, please?

Reply:
(Here and below)

Reply:
Yeah, I guess the diff shows the last PR plus this PR. This should go away once the last PR is merged. Please hold off reviewing this PR for now.